doc/source/tune/tutorials/tune-output.rst

```diff
@@ -240,13 +252,47 @@ You can do this in the trainable, as shown below:
                 step = result["training_iteration"]
                 library.log(res_dict, step=step)
 
+In the code snippet above, ``library`` refers to whatever 3rd party logging library you are using.
+Note that ``library.set_log_path(os.getcwd())`` is an imaginary API that we are using
+for demonstation purposes, and it highlights that the third-party library
+should be configured to log to the Trainable's *working directory.* By default,
+the current working directory of both functional and class trainables will be set to the
+corresponding trial directory once it's been launched as a remote Ray actor.
 
-Note: For both functional and class trainables, the current working directory is changed to something
-specific to that trainable once it's launched on a remote actor.
-
-In the distributed case, these logs will be sync'ed back to the driver under your logger path.
+When running with multiple nodes using the :ref:`default syncing method <tune-default-syncing>`,
+trial artifacts will be synchronized to the driver node under the specified path.
 This will allow you to visualize and analyze logs of all distributed training workers on a single machine.
 
+When :ref:`specifying a cloud upload directory <tune-cloud-checkpointing>`, trial artifacts will be uploaded to that cloud bucket
+for later analysis. Note that in this case, the driver node will not necessarily contain
+artifacts from *all* trials -- only the ones that were running on that node.
+To disable artifacts from being uploaded to the cloud, set ``SyncConfig(sync_artifacts=False)`` in :class:`~ray.tune.syncer.SyncConfig`.
+
+.. warning::
+
+    Appending to trial artifacts upon restoration is not currently supported.
+    To get around this, we recommend saving trial artifacts a separate files with unique filenames.
+
+    For example, instead of doing this:
+
+    .. code-block:: python
+
+        def appending_train_fn(config):
+            for i in range(config["num_epochs"]):
+                with open("./artifact.txt", "a") as f:
+                    f.write(f"Some data about iteration {i}\n")
+
+    You should log artifacts as independent files with unique filenames:
+
+    .. code-block:: python
+
+        def separate_files_train_fn(config):
+            for i in range(config["num_epochs"]):
+                with open(f"./artifact_{i}.txt", "w") as f:
+                    f.write(f"Some data about iteration {i}\n")
+
+    If you are running into issues with this, please `file an issue <https://github.com/ray-project/ray/issues>`_!
```

Comment:
```suggestion
    If you are running into issues, `file an issue <https://github.com/ray-project/ray/issues>`_!
```

Reply:
I am not familiar with the syntax of the underscore at the end of the line. I wanted to suggest that you remove the exclamation point.